src/utils/saveToCart.ts

```diff
@@ -13,12 +14,12 @@ const saveToCart = (game: Game) => {
       return {};
     }
     const updatedCart = [...parsedCart, game];
-    alert(`${game.name} added to cart`);
+    alertModal(`${game.name} added to cart`);
     dispatch(setCartState(updatedCart));
     return localStorage.setItem(`${username}Cart`, JSON.stringify(updatedCart));
```

Comment:
I see repetitive logic. Need to write code smarter. Please refactor saveToCart function